src/test/java/apoc/hashing/FingerprintingTest.java

```diff
@@ -0,0 +1,75 @@
+package apoc.hashing;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.harness.junit.Neo4jRule;
+import org.neo4j.helpers.collection.Iterators;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class FingerprintingTest {
+
+    @Rule
+    public Neo4jRule neo4j = new Neo4jRule()
```

Comment:
harness is really slow for tests (firing up a server)
please stick to the embedded db tests